doc/reference/configuration/configuration_reference.rst

```diff
@@ -214,13 +239,1030 @@ This section describes options related to :ref:`storing configuration in etcd <c
+
+.. confval:: <user_or_role_name>.privileges.sql
+
+
+
+
+..  _configuration_reference_database:
+
+database
+--------
+
+The ``database`` section defines database-specific configuration parameters, such as an instance's read-write mode, transaction isolation level, and so on.
+
+.. NOTE::
+
+    ``database`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`database.hot_standby <configuration_reference_database_hot_standby>`
+-   :ref:`database.instance_uuid <configuration_reference_database_instance_uuid>`
+-   :ref:`database.mode <configuration_reference_database_mode>`
+-   :ref:`database.replicaset_uuid <configuration_reference_database_replicaset_uuid>`
+-   :ref:`database.txn_isolation <configuration_reference_database_txn_isolation>`
+-   :ref:`database.txn_timeout <configuration_reference_database_txn_timeout>`
+-   :ref:`database.use_mvcc_engine <configuration_reference_database_use_mvcc_engine>`
+
+.. _configuration_reference_database_hot_standby:
+
+.. confval:: database.hot_standby
+
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_DATABASE_HOT_STANDBY
+
+
+.. _configuration_reference_database_instance_uuid:
+
+.. confval:: database.instance_uuid
+
+    An :ref:`instance UUID <replication_uuid>`.
+
+    By default, instance UUIDs are generated automatically.
+    ``database.instance_uuid`` can be used to specify an instance identifier manually.
+
+    UUIDs should follow these rules:
+
+    *   The values must be true unique identifiers, not shared by other instances
+        or replica sets within the common infrastructure.
+
+    *   The values must be used consistently, not changed after the initial setup.
+        The initial values are stored in :ref:`snapshot files <index-box_persistence>`
+        and are checked whenever the system is restarted.
+
+        .. TODO: https://github.com/tarantool/doc/issues/3661 mention that UUIDs can be dropped during migration.
+
+    *   The values must comply with `RFC 4122 <https://tools.ietf.org/html/rfc4122>`_.
+        The `nil UUID <https://tools.ietf.org/html/rfc4122#section-4.1.7>`_ is not allowed.
+
+    See also: :ref:`database.replicaset_uuid <configuration_reference_database_replicaset_uuid>`
+
+    |
+    | Type: string
+    | Default: :ref:`box.NULL <box-null>`
+    | Environment variable: TT_DATABASE_INSTANCE_UUID
+
+
+.. _configuration_reference_database_mode:
+
+.. confval:: database.mode
+
+    An instance's operating mode.
+    This option is in effect if :ref:`replication.failover <configuration_reference_replication_failover>` is set to ``off``.
+
+    The following modes are available:
+
+    -   ``rw``: an instance is in read-write mode.
+    -   ``ro``: an instance is in read-only mode.
+
+    If this option is not specified explicitly, its default value depends on the number of instances in a replica set. For a single instance, the ``rw`` mode is used, while for multiple instances, the ``ro`` mode is used.
```

Comment:
Suggestion: _If not specified explicitly, the default value depends on the number of instances in a replica set._

Reply:
Much better, thanks